hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/contract/s3a/ITestS3AContractRename.java

```diff
@@ -45,6 +46,12 @@ public class ITestS3AContractRename extends AbstractContractRenameTest {
   public static final Logger LOG = LoggerFactory.getLogger(
       ITestS3AContractRename.class);
 
+  @Override
+  public void setup() throws Exception {
+    super.setup();
+    skipIfAnalyticsAcceleratorEnabled(getContract().getConf());
```

Comment:
Rather than skipping, can we not configure to run all these tests with S3ASeekableStream S3AInputStream ?That will lead to huge test coverage for the new read flow.